packages/libs/eda/src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -1569,6 +1571,13 @@ function getRequestParams(
           xMin: config?.independentAxisRange?.min,
           xMax: config?.independentAxisRange?.max,
         }
+      : binWidth &&
```

Comment:
```suggestion
      // send back end a viewport to prevent edge-case 500s with single-valued variables when a binWidth is provided
      : binWidth &&
```

Reply:
Done!